src/pages/index.js

```diff
@@ -65,55 +47,74 @@ function handleEditProfileButtonClick() {
   formValidators["profile-form"].disableSubmitButton();
 }
 
-profilePopup.setEventListeners();
+function loadApiCards() {
+  api.getCards().then(data => {
+    data.forEach((datum) => (renderCard(datum)));
+  });
+}
 
-editProfileButton.addEventListener("click", handleEditProfileButtonClick);
+function handleDelButtonClick(card) {
+  deleteDialogue.open();
+  deleteIdentification = card.id;
+}
 
-function renderCard(card) {//to address the recursive code I had been writing in the section and addphoto initializations
-  const newCard = new Card(card, "#card", handlePhotoClick);
+function renderCard(card) {
+  const newCard = new Card(card, "#card", handlePhotoClick, handleDelButtonClick, handleLikeClick);//ammended the delete and like handler so it could tell the other popup to open when the Card's del button was pressed and utilize the api.  
   const newCardElement = newCard.generateCardElement();
   cardSection.addItem(newCardElement);
 }
 
-const addPopup = new PopupWithForm(addPhotoPopup, {
-  handleFormSubmit: ({ title, image_url: imageUrl }) => {
-    const newObj = {
-      name: title,
-      link: imageUrl
-    };
-    renderCard(newObj);    
-    formValidators["add-photo-form"].resetFormValidation();
-    addPopup.close();
-  },
-});
-
-
-addPopup.setEventListeners();
-
-const cardSection = new Section(
-{
-  items: initialCards,
-  renderer: renderCard,
-},
-".cards"
-);
-
-cardSection.renderItems();
+function handleDeleteSubmit() {
+  const deleteSubmit = document.querySelector("#delete-submit");
+  if (!api.isComplete) {
+    deleteSubmit.textContent = "Deleting...";
+  }
+  api.deleteCard(deleteIdentification);
+  cardSection.clear();  
+  deleteIdentification = "";
+  setTimeout(() => {
+    loadApiCards();
+    deleteDialogue.close();  
+  }, 700);  
```

Comment:
Every server request in `index.js` should look something like this:

```js
api.someMethod()
.then(res => {
// here is the response that should be used to update the DOM.
// also in `then` you should close the popup
})
.catch(err => // here you catch possible errors)
.finally(() => // here you bring the default button text back )
``` 

Please, correct it in the whole code